CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+- SONAR-13796 Display the full date when hovering a date from now
```

Comment:
Please just also list in the changelog the components that where dropped (and eventually by what they should be replaced)